support.rst

```diff
@@ -0,0 +1,85 @@
+.. class:: hide-rst-heading
+
+Support
+=======
+
+Please first read the docs, the existing issue tracker issues and mailing
+list posts -- a lot of stuff is already documented / explained / discussed /
+filed there.
+
+Issue Tracker
+-------------
+
+If you've found a bug or have a concrete feature request, please create a new
+ticket on the project's `issue tracker`_.
+
+For more general questions or discussions, IRC or mailing list are preferred.
+
+Chat (IRC)
+----------
+Join us on channel #borgbackup on chat.freenode.net.
+
+As usual on IRC, just ask or tell directly and then patiently wait for replies.
+Stay connected.
+
+You could use the following link (after connecting, you can change the random
+nickname you get by typing "/nick mydesirednickname"):
+
+http://webchat.freenode.net/?randomnick=1&channels=%23borgbackup&uio=MTY9dHJ1ZSY5PXRydWUa8
+
+.. _mailing_list:
+
+Mailing list
+------------
+
+To find out about the mailing list, its topic, how to subscribe, how to
+unsubscribe and where you can find the archives of the list, see the
+`mailing list homepage
+<https://mail.python.org/mailman/listinfo/borgbackup>`_.
+
+Twitter
+-------
+
+Follow @borgbackup for announcements. You can also add @borgbackup if you
+would like to get retweeted for a borg related tweet.
+
+Please understand that Twitter is not suitable for longer / more complex
+discussions - use one of the other channels for that.
+
+Bounties and Fundraisers
+------------------------
+
+We use `BountySource <https://www.bountysource.com/teams/borgbackup>`_ to allow
+monetary contributions to the project and the developers, who push it forward.
+
+There, you can give general funds to the borgbackup members (the developers will
+then spend the funds as they deem fit). If you do not have some specific bounty
+(see below), you can use this as a general way to say "Thank You!" and support
+the software / project you like.
+
+If you want to encourage developers to fix some specific issue or implement some
+specific feature suggestion, you can post a new bounty or back an existing one
+(they always refer to an issue in our `issue tracker`_).
+
+As a developer, you can become a Bounty Hunter and win bounties (earn money) by
+contributing to Borg, a free and open source software project.
+
+We might also use BountySource to fund raise for some bigger goals.
+
+.. _security-contact:
```

Comment:
this stuff is linked from elsewhere in the docs, so we need a solution for that.